web/src/lib/components/asset-viewer/photo-viewer.svelte

```diff
@@ -6,12 +6,14 @@
   import { notificationController, NotificationType } from '../shared-components/notification/notification';
   import { useZoomImageWheel } from '@zoom-image/svelte';
   import { photoZoomState } from '$lib/stores/zoom-image.store';
+  import { isImageWebCompatible } from '$lib/utils/asset-utils';
 
   export let asset: AssetResponseDto;
   export let element: HTMLDivElement | undefined = undefined;
 
   let imgElement: HTMLDivElement;
   let assetData: string;
+  let hasOriginalDataFetchingBeenTriggered = false;
```

Comment:
This is a bit verbose. Could you replace it with `fetchedOriginal`?

Reply:
I think `fetchedOriginal` is a bit confusing as this variable becomes true just before starting to fetch the data. So it can be true even if the data has not been fetched, when fetching is in progress. The idea is to prevent the download from starting more than once. What do you think about `originalDataFetchingTriggered`?

Reply:
Yeah, I can see how that could be confusing when debugging. 

I like `originalDataFetchingTriggered`: it's less wordy, but still conveys what it actually means.